pandas/core/indexes/datetimes.py

```diff
@@ -393,11 +393,19 @@ def _is_dates_only(self) -> bool:
         -------
         bool
         """
+        from pandas import Timedelta
```

Comment:
Can `datetime.timedelta` be used instead?

Reply:
Yes, that works too. Is this to reduce dependencies of modules between themselves in pandas?

Reply:
It's good to avoid runtime imports when possible